functions/src/utils/assert.js

```diff
@@ -18,5 +31,11 @@ const assertValidUrl = (url, paramName = "url") => {
   }
 };
 
+exports.assertValidAuth = assertValidAuth;
 exports.assertValidVenueId = assertValidVenueId;
 exports.assertValidUrl = assertValidUrl;
+
+/**
+ * @deprecated use assertValidAuth export/naming instead
+ */
+exports.checkAuth = assertValidAuth;
```

Comment:
[note] This is so I don't have to rename all of the other usages across the backend functions, which is OOS of this PR and can be done in a future cleanup.